src/app/client/src/app/modules/groups/components/activity/activity-search/activity-search.component.ts

```diff
@@ -127,29 +127,20 @@ export class ActivitySearchComponent implements OnInit, OnDestroy {
     });
   }
   private fetchContentOnParamChange() {
-    combineLatest([this.activatedRoute.params, this.activatedRoute.queryParams, this.groupsService.getGroupById(this.groupId, true, true)])
+    combineLatest([this.activatedRoute.params, this.activatedRoute.queryParams])
       .pipe(debounceTime(5), // to sync params and queryParams events
         delay(10), // to trigger pageexit telemetry event
         tap(data => {
           this.showLoader = true;
           // TODO set telemetry here
         }),
-        map((result) => ({ params: { pageNumber: Number(result[0].pageNumber) }, queryParams: result[1], group: result[2] })),
+        map((result) => ({ params: { pageNumber: Number(result[0].pageNumber) }, queryParams: result[1] })),
         takeUntil(this.unsubscribe$))
-      .subscribe(({ params, queryParams, group }) => {
-        const user = _.find(_.get(group, 'members'), (m) => _.get(m, 'userId') === this.userService.userid);
-
-        /* istanbul ignore else */
-        if (!user || _.get(user, 'role') === 'member' || _.get(user, 'status') === 'inactive' || _.get(group, 'status') === 'inactive') {
-          this.toasterService.warning(this.resourceService.messages.emsg.noAdminRole);
-          this.groupsService.goBack();
-        } else {
-          this.groupData = this.groupsService.addGroupFields(group);
+      .subscribe(({ params, queryParams }) => {
```

Comment:
Where did you handled this logic. Is it taken care by existing code

> 
>  if (!user || _.get(user, 'role') === 'member' || _.get(user, 'status') === 'inactive' || _.get(group, 'status') === 'inactive') {
>           this.toasterService.warning(this.resourceService.messages.emsg.noAdminRole);
>           this.groupsService.goBack();
>

Reply:
these scenarios are no longer applicable, currently, the user can not go inside the add activity page if 
1. he is a member
2. group is deactivated